website/for_teachers.py

```diff
@@ -653,14 +666,16 @@ def preview_teacher_mode(self):
         return redirect("/for-teachers")
 
     @route("/exit-preview-teacher-mode", methods=["GET"])
+    @route("/exit-preview-teacher-mode", methods=["GET"], subdomain="<subdomaon>")
```

Comment:
I think there is a typo: `"<subdomaon>"` should be `"<language>"`? Also, shouldn't there be a `language="en"` parameter to the exit_teacher_mode method?

Reply:
Good catch! Thank you so much